doc/classes/DisplayServer.xml

```diff
@@ -1033,6 +1033,13 @@
 				Returns the mouse cursor's current position in screen coordinates.
 			</description>
 		</method>
+		<method name="mouse_is_mode_async" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns if the current platform has its mouse mode set asynchronously.
```

Comment:
```suggestion
				Returns whether the current platform has its mouse mode set asynchronously. See also [method mouse_set_mode].
```

Sorry I only just noticed another improvement here ^^

Should be either `Returns [code]true[/code] if ...` or `Returns whether ...`

Also add reference to `mouse_set_mode`.